docs/testing.md

```diff
@@ -39,20 +39,54 @@ They generally follow TPCH but are not guaranteed to be the same.
 
 ## Unit tests
 
-Unit tests exist in the tests directory. This is unconventional and is done so we can run the tests
-on the final shaded version of the plugin. It also helps with how we collect code coverage.
+Unit tests exist in the [here](tests) directory. This is unconventional and is done so we can run
```

Comment:
"unit tests exist in the here directory" is not very readable.  Also not sure why you are linking to.  There is no tests directory or file under docs that this can or should link to.